src/test/kotlin/no/nav/tilleggsstonader/sak/behandling/OpprettBehandlingServiceIntegrationTest.kt

```diff
@@ -155,17 +165,47 @@ class OpprettBehandlingServiceIntegrationTest : IntegrationTest() {
         internal fun `opprettBehandling av revurdering er mulig hvis det finnes en førstegangsbehandling på vent`() {
             every { unleashService.isEnabled(Toggle.KAN_HA_FLERE_BEHANDLINGER_PÅ_SAMME_FAGSAK) } returns true
             val fagsak = testoppsettService.lagreFagsak(fagsak())
-            testoppsettService.lagre(behandling(fagsak, BehandlingStatus.SATT_PÅ_VENT))
+            val førstegangsbehandling = testoppsettService.lagre(behandling(fagsak))
+            testoppsettService.ferdigstillBehandling(førstegangsbehandling)
+
+            testoppsettService.lagre(behandling(fagsak, BehandlingStatus.SATT_PÅ_VENT, type = BehandlingType.REVURDERING))
+
             // Sjekker at denne ikke kaster feil
             opprettBehandlingService.opprettBehandling(
                 OpprettBehandling(
                     fagsak.id,
                     behandlingsårsak = behandlingÅrsak,
                     oppgaveMetadata = opprettBehandlingOppgaveMetadata,
+                    tillatFlereÅpneBehandlinger = true,
                 ),
             )
```

Comment:
Burde vi hatt en slik assertion her og? For å tydliggjøre at den nye behandlingen blir satt rett på vent - for det er det som skjer?

`            assertThat(nyBehandling.status).isEqualTo(BehandlingStatus.SATT_PÅ_VENT)
`